IDL/final.idl

```diff
@@ -97,3 +97,8 @@ struct FinalInheritanceStruct : FinalShortStruct
 struct InheritanceEmptyStruct : FinalShortStruct
 {
 };
+
+struct FinalExtensibilityInheritance : FinalShortStruct
+{
+    long var_long;
+};
```

Comment:
Is this structure meant to be appendable or final? As I have previously suggested, I think explicitly stating the extensibility is for the best.

It might be interesting to keep this structure but also define the final one following the specification XTypes v1.3 clause 7.2.2.4.5

> A Structure Type may inherit from another Structure Type as long as the following conditions are met:
>
> * The derived structure either has the extensibility kind unspecified, or else, it has the same extensibility kind as the base type.

This suggestion applies to the structures added into the other extensibility IDLs.

Reply:
The case you are talking about is already tested. This case tests the inheritance when a derived structure 
does not have the extensibility defined but the base structure does.